handbook/people-ops/expenses.md

```diff
@@ -28,9 +28,20 @@ We use Expensify for expense report submission and approval. You will receive an
 - Add a description of your expense in the designated field. 
 
 
-### Expense categories
+### When submitting an expense
 
-Please ensure that each expense is properly categorized. Some frequently asked questions about categories include:
+- Please ensure that each expense is properly categorized. Some frequently asked questions about categories include:
+  - Software purchases or subscriptions should be categorized as "Software and SaaS <$1000"
+  - Computer purchases and other hardware (monitor, headphones, etc.) should be categorized as "Computer & Related <$1000" 
+  - Home office set-up purchases (chairs, desks, etc.) should be categorized as "Furniture & Office"
+- Please ensure the date and description fields are properly filled out
+  - The date listed on your expense should be the date of purchase (not the date of submission).
+  - The description field should list exactly what item is being expensed.
 
-- Software purchases or subscriptions should be categorized as "Software and SaaS <$1000"
-- Wellness/gym reimbursements should be categorized as "Fringe benefits"
+### Internet, phone, and wellness benefits
+- For US based teammates: these fringe benefits are automatically added to your paycheck (the total amount is split up over both paychecks for the month), there is no need to submit them via Expensify. 
+- For teammates based outside the US: you can have have these as fixed line items on your invoices and you will be paid for the full amount every month.
+- Fringe benefit amount (per month):
+  - Internet: $50
+  - Phone: $75
+  - Wellness: $100
```

Comment:
Just confirming -  for US based folks, this is changing from $85/month to $100/month and also, it is automatically being added to paycheck (I have still been submitting expenses for wellness, but this is no longer required?).

Reply:
Yep, this is changing for everyone– we bumped it up to $100 and the main difference here is that US folks no longer have to expense it (non-US folks still have to add a line item on their invoices). This change won't take effect as of the 12/15 pay cycle, but I wanted to get this merged before announcing so I can link to it!

Reply:
🎉  awesome!